viewer/packages/3d-overlays/src/Overlay3DIcon.ts

```diff
@@ -127,9 +129,13 @@ export class Overlay3DIcon<ContentType = DefaultOverlay3DContentType> implements
     }
   }
 
-  setColor(color: Color): void {
-    this._color = color;
-    this._events.parametersChange.fire({ color, visble: this.getVisible() });
+  setColor(color: Color | undefined): void {
```

Comment:
This feels more typescripty to me. Doing a naked `setColor();` which then adds some random color (usually white?) seems a little off for me. setColor('default'), at least makes it explicit that some default color will be applied.
```suggestion
  setColor(color: Color | 'default'): void {
```